packages/manager/modules/adp/src/deploy/regions.html

```diff
@@ -0,0 +1,45 @@
+<oui-tabs class="mb-xs-2 mb-4">
+    <!-- all regions -->
+    <oui-tabs-item data-heading="{{ 'adp_deploy_all_regions' | translate }}">
+        <div class="d-flex d-sm-flex mx-3" style="flex-wrap: wrap;">
```

Comment:
https://getbootstrap.com/docs/4.1/utilities/flex/
```suggestion
        <div class="d-flex d-sm-flex flex-wrap mx-3">
```